kratos-nextjs-tailwind/config/kratos.yml

```diff
@@ -0,0 +1,140 @@
+version: v0.7.1-alpha.1
+serve:
+  public:
+    base_url: http://127.0.0.1:4433/
+    cors:
+      allowed_origins:
+        - http://127.0.0.1:3000
+      allowed_methods:
+        - POST
+        - GET
+        - PUT
+        - PATCH
+        - DELETE
+      allowed_headers:
+        - Authorization
+        - Cookie
+        - Content-Type
+        - X-Session-Token
+      exposed_headers:
+        - Content-Type
+        - Set-Cookie
+      allow_credentials: true
+      options_passthrough: true
+      debug: true
+      enabled: true
+  admin:
+    base_url: http://127.0.0.1:4434/
+    host: "0.0.0.0"
+    port: 4434
+
+selfservice:
+  default_browser_return_url: http://127.0.0.1:3000/
+  whitelisted_return_urls:
+    - http://127.0.0.1:3000/
+  methods:
+    oidc:
+      enabled: true
+      config:
+        providers:
+          - id: github
+            provider: github
+            client_id: ... # Add the client ID
+            client_secret: ... # Add the client secret
+            mapper_url: file:///etc/config/kratos/oidc.github.jsonnet
+            scope:
+              - user:email
+          - id: google # this is `<provider-id>` in the Authorization callback URL. DO NOT CHANGE IT ONCE SET!
+            provider: google
+            client_id: ... # Add the client ID
+            client_secret: ... # Add the client secret
+            mapper_url: file:///etc/config/kratos/oidc.google.jsonnet
+            scope:
+              - email
+              - profile
+            requested_claims:
+              id_token:
+                email:
+                  essential: true
+                email_verified:
+                  essential: true
+                given_name:
+                  essential: true
+                family_name: null
+                hd: null # If you want the G Suite domain
+    password:
+      enabled: true
+
+  flows:
+    error:
+      ui_url: http://127.0.0.1:3000/error
+
+    settings:
+      ui_url: http://127.0.0.1:3000/settings
+      privileged_session_max_age: 15m
+      after:
+        default_browser_return_url: http://127.0.0.1:3000/welcome
+
+    recovery:
+      enabled: true
+      ui_url: http://127.0.0.1:3000/recovery
+      after:
+        default_browser_return_url: http://127.0.0.1:3000/welcome
+
+    verification:
+      enabled: true
+      ui_url: http://127.0.0.1:3000/verify
+      after:
+        default_browser_return_url: http://127.0.0.1:3000/welcome
+
+    logout:
+      after:
+        default_browser_return_url: http://127.0.0.1:3000/
+
+    login:
+      ui_url: http://127.0.0.1:3000/
+      lifespan: 10m
+      after:
+        default_browser_return_url: http://127.0.01:3000/welcome
+
+    registration:
```

Comment:
Add this to set a session if social registartion is done :
 after:
        oidc:
          hooks:
            - hook: session